crates/uv-distribution-filename/src/wheel.rs

```diff
@@ -104,6 +108,14 @@ impl WheelFilename {
         )
     }
 
+    /// Returns a consistent identifier useful where full stem might be too long.
+    /// format: `{truncated(version-tags)}-{digest}`
+    pub fn stem_identifier(&self) -> String {
```

Comment:
Couldn't think of a better name for this ;(
Feel free to change it or reject this commit :)